tools/chplcheck/src/lsp.py

```diff
@@ -93,7 +81,7 @@ def build_diagnostics(uri):
             diagnostics = []
             for (node, rule) in driver.run_checks(context, asts):
                 diagnostic = Diagnostic(
-                    range= location_to_range(node.location()),
+                    range=chapel.lsp.location_to_range(node.location()),
```

Comment:
Nitpickiest of all nitpicks -- there's a space on `range=` elsewhere, but not here. Pick one? I don't mind either way.